devfiles/latest/che-idea/latest.yaml

```diff
@@ -24,6 +24,8 @@ components:
           path: /projector
         - name: projector-configuration
           path: /home/user/.jetbrains
+        - name: projector-java-configuration
```

Comment:
what is worrying me is that if we've a docker container which is not using /home/user it may probably not work
so maybe need to use a dedicated path independent of the HOME value

Reply:
for a standalone docker container there should be a dedicated volume mount provided by default here: https://github.com/che-incubator/jetbrains-editor-images/blob/main/projector.sh#L22

Reply:
it's not a standalone docker container.

You can start a che workspace with a container that is not using the universal developer image

Reply:
IntelliJ IDEA uses standard `FileSystemPreferences` class from JDK which relies to "user.home"... So it's not easy to move that folder to another location:

https://github.com/frohoff/jdk8u-jdk/blob/master/src/solaris/classes/java/util/prefs/FileSystemPreferences.java#L104-L121

Reply:
We can change the value of "user.home", but I'm not think that this is a nice solution.